src/foam/mlang/mlang.js

```diff
@@ -317,7 +317,8 @@ foam.CLASS({
       return foam.String.constantize(this.cls_.name) + '(' +
           this.arg1.toString() + ', ' +
           this.arg2.toString() + ')';
-    }
+    },
+    function contradicts(other) { return false; }
```

Comment:
should this be a more general simplification method (that AND could call, as here, but returns a new simpler mlang rather than just bool).